deps.sh

```diff
@@ -5,7 +5,6 @@ set -e
 
 # Test dependencies:
 tarantoolctl rocks install luatest 0.5.3
-tarantoolctl rocks install luacheck 0.25.0
```

Comment:
Do we really need this change?

Reply:
`deps.sh` used in YAML files for GH Actions for dependencies installation. With my proposed patch luacheck is unused in tasks where crud tested. In a new pipeline luacheck installed directly using `tarantoolctl`, using `deps.sh` I believe will take more time due to `cartridge` and `luatest`.

Reply:
So the answer is "Yes".

Reply:
This change is not a part of GH actions primary. This file should make life simple for developers.

For local development you need to call ./deps.sh - and you will get all needed toolchain for local testing. Please revert this change.

Reply:
reverted